dietpi/func/dietpi-banner

```diff
@@ -248,9 +248,9 @@ $GREEN_LINE"
 		# DietPi-VPN connection status
 		(( ${aENABLED[13]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[13]} $GREEN_SEPARATOR $(/boot/dietpi/dietpi-vpn status 2>&1)"
 		# Freespace (RootFS)
-		(( ${aENABLED[7]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[7]} $GREEN_SEPARATOR $(df -h --output=avail / | mawk 'NR==2 {print $1}' 2>&1)"
+		(( ${aENABLED[7]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[7]} $GREEN_SEPARATOR $(df -h / | mawk 'NR==2 {printf "%s / %s (%s)", $4, $2, $5}' 2>&1)"
```

Comment:
Not really shorter or so, but I like to be explicit about the processed fields:
```suggestion
		(( ${aENABLED[7]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[7]} $GREEN_SEPARATOR $(df -h --output=avail,size,pcent / | mawk 'NR==2 {printf $1" / "$2" ("$3"%)"}' 2>&1)"
```

Reply:
Good idea!
Ive updated the PR + in the end I started to think that showing the used percentage might be a bit confusing, since you have displaying the free space / total space, so I ended up removing it...